### ISSUE TITLE
Travis.ci : No output has been received in the last 10 minutes

### ISSUE THREAD

**Issue by gillesbourgeat** [thelia/thelia#1612]
Travis is currently slow.
The step "Creating categories and products" does not give an output fast enough.
Of that, travis stops the test

```
Clearing tables
Creating customers
Creating features
Creating attributes
Creating templates
Creating folders and contents
Creating brands
Creating categories and products

No output has been received in the last 10 minutes, this potentially indicates a stalled build or something wrong with the build itself.

The build has been terminated
```

**Comment by lunika** [comment]
Yes travis is very slow, but it's a free service, lot of opensource software use it and they have to share all the resources between all of this project.

You can also try the container based version of travis http://docs.travis-ci.com/user/workers/container-based-infrastructure/ but for this you have to disable ```sudo``` and without sud you can't install needed packages.

Maybe you can try an other service like https://circleci.com/ or try to launch the test suite in scrutinizer

**Comment by roadster31** [comment]
We could also create less products and categories.

**Comment by lovenunu** [comment]
Since we have a dedicated VM for Thelia, we could run the tests suite in a Jenkins too.

**Comment by lunika** [comment]
@lovenunu yes but using a SAAS solution is easier for the team because we have less thing to maintain, we don't have to increase the wheel once again etc

@roadster31 this can be a solution

@gillesbourgeat Travis just publish a new documentation for the container based architecture : http://docs.travis-ci.com/user/migrating-from-legacy/?utm_source=legacy-notice&utm_medium=banner&utm_campaign=legacy-upgrade

**Comment by roadster31** [comment]
I've added a way to speed up the faker in #1615, by switching off the real text generation.

**Comment by bibich** [comment]
Fixed by #1615
